File: PR1/PR1_Ej1/src/FactoriaMontania.java
public class FactoriaMontania implements FactoriaCarreraYBicicleta{
    @Override
    public Carrera crearCarrera() {
        return new CarreraMontania();
    }

    @Override
    public Bicicleta crearBicicleta(int id) {
        return new BicicletaMontania(id);
    }
}

File: PR1/PR1_Ej2/FactoriaMontania.py
from FactoriaCarreraYBicicleta import FactoriaCarreraYBicicleta
from CarreraMontania import CarreraMontania
from BicicletaMontania import BicicletaMontania
class FactoriaMontania(FactoriaCarreraYBicicleta):

    def crearCarrera(self):
        return CarreraMontania()

    def crearBicicleta(self):
        return BicicletaMontania()